.README/README-KO.md

```diff
@@ -0,0 +1,48 @@
+<p align="center">
+  <img
+    src="https://raw.githubusercontent.com/rust-analyzer/rust-analyzer/master/assets/logo-wide.svg"
+    alt="rust-analyzer logo">
+</p>
+
+<p align="center">
+  <a href="https://github.com/rust-lang/rust-analyzer/blob/master/README.md">en</a>
+  <a href="#">ko</a>
+</p>
+
+러스트 분석기는 러스트 언어를 위한 모듈형 컴파일러 프론트엔드입니다.
```

Comment:
In the Description and License sections you've kept the English/Latin spelling of "rust-analyzer"; same here for "Rust", perhaps. Should it be spelled the same way here? No strong opinion.